pyvecorg/views.py

```diff
@@ -11,23 +11,37 @@
 data = load_data()
 
 
-@app.route('/')
-def index_redirect():
-    if request.accept_languages.best_match(['en', 'cs', 'sk']) == 'en':
-        return redirect(url_for('index', lang='en'))
-    return redirect(url_for('index', lang='cs'))
-
-
 @app.route('/favicon.ico')
 def favicon():
     static = os.path.join(app.root_path, 'static')
     return send_from_directory(static, 'favicon.ico',
                                mimetype='image/vnd.microsoft.icon')
 
 
+@app.route('/')
+def index_redirect():
+    return redirect(url_for('index', lang=detect_lang(request)))
+
+
 @app.route('/<lang>/')
 def index(lang):
     context = select_language(data, lang)
     context['lang'] = lang
     context['now'] = datetime.now()
     return render_template('index.html', **context)
+
+
+@app.route('/api.json')
+def api_redirect():
+    return redirect(url_for('api', lang=detect_lang(request)))
```

Comment:
This doesn't actually work for a static page.
You're aware of that, right?

Reply:
Good catch. :)